bundle/manifests/saas-operator-saas-operator_v1_serviceaccount.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  creationTimestamp: null
+  name: saas-operator-saas-operator
```

Comment:
Is this name correct? seems to be `saas-operator` x2

Reply:
This file is generated by kustomize, when adding the prefix, but really looks messy. I will see how we can avoid this behaviour.

Reply:
Fixed in https://github.com/3scale/saas-operator/pull/52/commits/74c92848b9854f08f7e975c9e8e22e74af2a61b9